destination/destination.go

```diff
@@ -91,6 +94,8 @@ func (d *Destination) WriteAsync(ctx context.Context,
 		return d.err
 	}
 
+	// haris: thinking out loud, it might make sense to just put in a new line
+	// when there is no payload.
```

Comment:
For Newline we should ideally get an empty array [] in payload, which signifies empty rows. If we add an empty payload as Newline as well, that might lead to some confusion. Also, an empty payload is not a valid JSON, so we will have to have a default payload to replace it.
But, if it makes more sense for the conduit to add empty line, we can do that.

Reply:
It's actually more about the connector and what makes sense for the 3rd party system. But we can probably leave it as it is, and gauge the community's feedback about this.: )